database/src/main/resources/db/migration/V81__grunnlag.sql

```diff
@@ -0,0 +1,15 @@
+create table if not exists grunnlag_uføre
+(
+    id uuid primary key,
+    opprettet timestamptz not null,
+    fom date,
+    tom date,
+    uføregrad integer not null,
+    forventetInntekt integer not null
+);
+
+create table if not exists behandling_grunnlag
+(
+    behandlingId uuid not null,
```

Comment:
Ingen referanseintegritet (brukes både for revurdering og søknadsbehandling). Kan løses ved å innføre `revurdering_grunnlag` + `søknadsbehandling_grunnlag` evt kunne denne tabellen hatt egne kolonner for `søknadsbehandlingId` + `revurderingId` som kunn gjort denne sjekken.